api/libproject/devstart.rb

```diff
@@ -2016,7 +2016,7 @@ def invalidate_rdr_export(cmd_name, *args)
   end
 
   get_user_confirmation(
-    "RDR export invalidation should rarely be used; for backfilling workspace data you " +
```

Comment:
I think this is new true for both workspaces and users - will get confirmation elsewhere